src/tactic/unfold_cases.lean

```diff
@@ -0,0 +1,172 @@
+/-
+Copyright (c) 2020 Dany Fabian. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Dany Fabian
+-/
+
+import tactic.split_ifs
+namespace tactic
+
+
+
+open expr
```

Comment:
This and the suggestion below address [this comment](https://github.com/leanprover-community/mathlib/pull/3396/files#r454670051).
```suggestion
```